[PNRv2] - Billing and services updates (#7578)
ovh/docs: pages/account_and_service_management/managing_billing_payments_and_services/add_cloud_credit_to_project/guide.es-es.md

## Requisitos

> - Un proyecto de [Public Cloud](https://www.ovhcloud.com/es-es/public-cloud/) en su cuenta de OVHcloud

Reviewer: ```suggestion
- Un [proyecto de Public Cloud](https://www.ovhcloud.com/es-es/public-cloud/) en su cuenta de OVHcloud
```

> - Un proyecto de [Public Cloud](https://www.ovhcloud.com/es-es/public-cloud/) en su cuenta de OVHcloud

Reviewer: ```suggestion
- Un proyecto de [Public Cloud](/links/public-cloud/public-cloud) en su cuenta de OVHcloud
```

> - Accesar a su [Panel de configuración de OVHcloud](/links/manager)

Reviewer: ```suggestion
- Accesar a su [área de cliente de OVHcloud](/links/manager)
```

> - Tienes acceso a tu [Panel de configuración de OVHcloud](/links/manager)

Reviewer: ```suggestion
- Accesar a su [área de cliente de OVHcloud](/links/manager).
```

## Procedimiento

> Inicie sesión en el [Panel de configuración de OVHcloud](/links/manager), acceda a la sección `Public Cloud`{.action} y seleccione el proyecto de Public Cloud correspondiente. Abra `Credit and Vouchers`{.action} en el menú de la izquierda.

Reviewer: ```suggestion
Inicie sesión en su [área de cliente de OVHcloud](/links/manager), acceda a la sección `Public Cloud`{.action} y seleccione el proyecto de Public Cloud correspondiente. Abra `Credit and Vouchers`{.action} en el menú de la izquierda.
```
